src/web/components/ApiKeyManagement/KeyItem.tsx

```diff
@@ -42,6 +50,13 @@ function KeyItem({ apiKey: apiKeyInitial, onEdit, onDisable, availableRoles }: K
       {availableRoles.length > 0 && (
         <td className='action'>
           <div className='action-cell'>
+            {keyNeedsRotating && (
+              <FontAwesomeIcon
+                icon='triangle-exclamation'
+                title='We recommend you rotate your API key every year.'
```

Comment:
Perhaps we could use a `<Tooltip>` here for the message rather than relying on the title?

Reply:
True - changed